src/Components/test/testassets/ComponentsApp.Server/Pages/_Host.cshtml

```diff
@@ -12,7 +12,7 @@
     <link href="css/site.css" rel="stylesheet" />
 </head>
 <body>
-    <app>@(await Html.RenderComponentAsync<App>(new { Name = "Guest" }))</app>
+    <app>@(await Html.RenderComponentAsync<App>(new { Name = "Guest" }, RenderMode.Server))</app>
```

Comment:
Per our discussion, I would expect this to throw because it's non-static and it takes arguments.

Reply:
Per our dicussion, I would expect the arguments to be the second argument. Yeah, that's meta.

Reply:
Yes, I haven't made the tests work yet, so I just made whatever necessary for them to compile. I've added a check to throw if you pass arguments.

Reply:
Updated to make the tests work.